kalite/distributed/features/superuser_create.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Create superuser from the in browser modal
+
+    Scenario: Superuser already exists
+        Given there is superuser 
+        And I am on the homepage
+        Then there should be no modal displayed
+
+    Scenario: Superuser does not exist yet
+        Given there is no superuser
+        And I am on the homepage
+        Then I should see a modal
+        And I will not be able to dismiss the modal until a superuser is created
+
+    Scenario: Create superuser with empty username
+        Given the username is empty
+        When I click the create button
+        Then the username border will turn red
+        And the modal won't dismiss
+
+    Scenario: Create superuser with string longer than 30
```

Comment:
It's probably good to create one behave test for invalid input conditions, but for the specifics (i.e., password length requirements, valid emails) we can test the view function directly without the extra browser test overhead.